src/Arbor/Api/Gateway/RestGateway.php

```diff
@@ -302,6 +302,10 @@ public function bulkCreate(string $resource, Collection $collection, bool $check
         }
 
         if (isset($exception)) {
```

Comment:
This entire block should be refactored into the above catch block to remove the need for the `if (isset($exception)) {` block.

Reply:
@AnthonyPorthouse This part is refacoted now, so can you check PR now